README.md

```diff
@@ -1,3 +1,7 @@
 # simple-taiko-node
 
 Get started with the [guide](https://taiko.xyz/docs/guides/run-a-node).
+
+or use one line command
+
+# git clone https://github.com/tudorpintea999/taikonode.git && cd taikonode && sudo chmod +x taikoinstaller.sh && sudo /bin/bash taikoinstaller.sh
```

Comment:
why do we need taikoinstaller.sh in this repo if it is already inside of the cloned repo? can we remove it from here?

otherwise this looks okay to me, but we'll have to add a small warning about this being a community maintained software because of the possibility of updating the repo and changing the code over time (we can't audit PRs in the future)

Reply:
no i mean that in this PR we are adding a new file: `taikoinstaller.sh`. do we need `taikoinstaller.sh` if the user is just going to execute this command: `git clone https://github.com/tudorpintea999/taikonode.git && cd taikonode && sudo chmod +x taikoinstaller.sh && sudo /bin/bash taikoinstaller.sh`

won't this command use the `taikoinstaller.sh` that is pulled in as a part of cloning your repo? also is it possible to cleanup this command and make it something like: `curl -L https://gimmenode.xyz | bash` (inspired by https://book.getfoundry.sh/getting-started/installation#using-foundryup haha).

Reply:
i got it now, i did not knew where to add only that command in the repo, tell me where to add it in the repo and i ll also clean it up, thanks

Reply:
where to add what into the repo? i mean we should:
1. remove the script `taikoinstaller.sh` from this PR
2. [optional] serve the taikoinstaller.sh script over some website like https://gimmenode.vercel.app and then pipe that into | bash. 

the 2. is totally optional, i just think it's a bit cleaner. before we accept this PR however we must add a **warning** that this is third-party software and the user is trusting that the script is not stealing their private keys etc. lol

Reply:
where to add what into the repo? the actual command , `curl node.vercel.app/taiko | bash` , or you ll add that in the readme? let me create an account on vercel then i ll put here the working command

Reply:
hello, tried to deploy it via vercel but Build Failed
Your deployment's build step did not complete within the maximum of 45min
so for the moment i ll let it as it is in git. 

this is the final command 
`git clone https://github.com/tudorpintea999/TaikoNodeAutomation.git && cd TaikoNodeAutomation && sudo chmod +x taikoinstaller.sh && sudo /bin/bash taikoinstaller.sh`

Reply:
but i'm still wondering can't we remove the script in here? otherwise we have two different places where `taikoinstaller.sh` is maintained. it's not DRY

Reply:
i tought i removed it , it s now removed

Reply:
looks like my brain froze4 days ago, i wanted to remove the bash script and i removed the readme :(